app/src/org/commcare/connect/ConnectDateUtils.kt

```diff
@@ -20,6 +19,15 @@ object ConnectDateUtils {
         }
     }
 
+    val completedJobDateFormat: DateFormat =
+        SimpleDateFormat("dd MMM, yyyy", Locale.getDefault())
+
+    fun formatDateForCompletedJob(date: Date): String {
+        synchronized(completedJobDateFormat) {
+            return completedJobDateFormat.format(date)
+        }
+    }
```

Comment:
I don't think there is a particular reason we should be using 2 different date formats here and instead the guiding product principle here should be _**consistency**_ .  

I think product just missed out on this inconsistency but will support us uisng the same date format across two very similar widgets.

Reply:
I went ahead and did similar date formatting logic for the app home pages too for consistency with the progress pages - 4d776b18afd62e9bf4df8aae59dc91bf4dfa9652